doc/command_ref.rst

```diff
@@ -894,6 +894,7 @@ Mark Command
 Module Command
 ---------------
 
+Overview about Modularity in DNF is available at :ref:`man page dnf.modularity(7) <modularity-label>`
```

Comment:
I'd say:
> General Modularity overview is available at :ref:`dnf.modularity(7) man page <modularity-label>`.

(Don't forget the dot at the end)

The overview is not really specific to DNF AFAICS, the DNF specifics are actually here, i.e. the commands to work with it.

Reply:
Dots were added, DNF was removed. @lukash Thanks for information.